web-client/scripts/build.sh

```diff
@@ -1,17 +1,39 @@
+#!/bin/bash
+
 set -e
 
 # TODO: Disable rust compiler warnings with [lints] table in Cargo.toml when it becomes stable in 1.75 (https://github.com/rust-lang/cargo/issues/12115)
 
+# Defaults
 CARGO_PROFILE="release-wasm"
 CARGO_TARGET="wasm32-unknown-unknown"
 CARGO_OUTPUT="../target/$CARGO_TARGET/$CARGO_PROFILE/nimiq_web_client.wasm"
+TARGETS="bundler,web,nodejs"
+RUN_WASM_OPT=true
+BUILD_TYPES=true
+BUILD_LAUNCHER=true
+
+# Parse command line arguments - all arguments are optional
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --cargo-profile) CARGO_PROFILE="$2"; shift ;;
+        --cargo-target) CARGO_TARGET="$2"; shift ;;
+        --cargo-dir) CARGO_OUTPUT="$2/nimiq_web_client.wasm"; shift ;;
+        -o|--only) TARGETS="$2"; shift ;;
+        --skip-wasm-opt) RUN_WASM_OPT=false ;;
+        --skip-types) BUILD_TYPES=false ;;
+        --skip-launcher) BUILD_LAUNCHER=false ;;
+        *) echo "Unknown argument: $1"; exit 1 ;;
+    esac
+    shift
+done
 
 # Compile web-client to WASM for a given feature set
 # $1: features to enable
 function compile() {
     FEATURES=$1
 
-    printf "Compiling WASM for $FEATURES...\n"
+    echo "Compiling WASM for $FEATURES...\n"
```

Comment:
Here and below: If you change `printf` to `echo`, you need to remove the final `\n`.